deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/worker/DeployTagWorker.java

```diff
@@ -165,6 +168,7 @@ private void processBatch() throws Exception {
     @Override
     public void run() {
         try {
+            TimeUnit.MINUTES.sleep(2);
```

Comment:
As we intend to depend on the tags sent by Deployd, and since Deployd runs every minute, we believe it would be prudent to wait for about 2 minutes. This would help us prevent querying the CMDB when the tags are absent.

Reply:
can we change the freq of this worker instead of sleep here?

Reply:
Thank you for your input. I've reviewed the frequency of the tag-worker, which is set at 5 minutes; this appears to be acceptable. Regarding the waiting time, if we neglect to include it, there could be potential issues. For instance, if a deployment constraint is created, the CMDB query may precede the Deployd operation. This premature querying could create issues if we don't incorporate a waiting period.

Reply:
how waiting 2mins can help with that. deployment constraints and cmdb query may happen while waiting and before deployd operation?

Reply:
This brief pause allows deployd adequate time to send the tags to TAS and update them accordingly. Consequently, when the worker checks for the existence of the tags, the system would have been updated and will eliminate the need to query cmdb any further.